pyproject.toml

```diff
@@ -138,6 +138,7 @@ types-psutil = "^5.9.5.16"
 types-psycopg2 = "^2.9.21.14"
 cryptography = "^41.0.7"
 google-api-python-client = ">=1.7.11"
+sqlglot = {extras = ["rs"], version = "^20.11.0"}
```

Comment:
kick this out if not used!

Reply:
resolved in 1245d6d9d00d62b660af36e396fcb0524ad42c83